packages/fast-layouts-react/src/page/page.props.ts

```diff
@@ -14,6 +15,12 @@ export interface PageHandledProps extends PageManagedClasses {
      * The max width of the page
      */
     maxWidth?: string;
+
+    /**
+     * The CSS grid display, "grid" when true and "-ms-grid" when false
```

Comment:
I'm not following this comment - what is "true" or "false" here?

Maybe change to "The value assigned to the CSS display property"

Reply:
I'll update to that 👍 (The value assigned to the CSS display property. The property is assigned "grid" when true and "-ms-grid" when false.